### PR TITLE
Streamlined Top-Level Menus For Copter, Plane, and Rover. Rewrote Copter Intro and Wiki Readme

### PR DESCRIPTION
**Pull request by brianmusslewhite**
I've streamlined the top-level menus for Copter, Plane, and Rover, creating a unified structure. The Copter intro page and wiki README have also been updated. Any suggestions, questions, or feedback you might have are welcomed.

### CODE REVIEW

**Comment by Hwurzburg** [comment]
@brianmusslewhite Brian, thanks for this....a massive reorg like this will take some time to carefully review and check.....I am not opposed to the front end squash, although it does take more clicks to get to some topics and we will need to make sure that were are not hiding some topics too far down....

a few things right off the bat:
- there are many build errors that CI is not catching....the title underlines MUST be the same lengths as the titles
- you have created this off your master branch instead of a local working branch so no one can make corrections by pushing to this PR

**Comment by Hwurzburg** [comment]
list of current build errors, EXCLUDING the many title underline/overline errors mentioned above:
- /home/henry/Desktop/ardupilot_wiki/antennatracker/source/index.rst:42: WARNING: toctree contains reference to nonexisting document 'docs/common-table-of-contents'
- /home/henry/Desktop/ardupilot_wiki/planner/source/index.rst:29: WARNING: toctree contains reference to nonexisting document 'docs/common-table-of-contents'
- home/henry/Desktop/ardupilot_wiki/rover/source/docs/common-all-vehicle-types.rst: WARNING: document isn't included in any toctree
- /home/henry/Desktop/ardupilot_wiki/rover/source/docs/ready-to-use-rovers.rst: WARNING: document isn't included in any toctree

File: copter/source/_build/html/_sources/docs/flight-preparation-and-operation.rst.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+================
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply copter/source/_build/html/_sources/docs/flight-preparation-and-operation.rst.txt:1]: it also is missing the tag at the top:   _flight-preparation-and-operation:

File: copter/source/docs/troubleshooting-and-support.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+================
```
Comment (Hwurzburg) [inline copter/source/docs/troubleshooting-and-support.rst:1]: missing tag

File: plane/source/docs/flight-preparation-and-operation.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
+==========
```
Comment (Hwurzburg) [inline plane/source/docs/flight-preparation-and-operation.rst:1]: missing tag

File: plane/source/docs/further-information.rst [hunk 4]
```diff
@@ -0,0 +1,10 @@
+==========
```
Comment (Hwurzburg) [inline plane/source/docs/further-information.rst:1]: missing tag

File: plane/source/docs/plane-variants.rst [hunk 5]
```diff
@@ -0,0 +1,10 @@
+==========
```
Comment (Hwurzburg) [inline plane/source/docs/plane-variants.rst:1]: missing tag

File: plane/source/docs/setup.rst [hunk 6]
```diff
@@ -0,0 +1,10 @@
+==========
```
Comment (Hwurzburg) [inline plane/source/docs/setup.rst:1]: missing tag

File: plane/source/docs/troubleshooting-and-support.rst [hunk 7]
```diff
@@ -0,0 +1,11 @@
+==========
```
Comment (Hwurzburg) [inline plane/source/docs/troubleshooting-and-support.rst:1]: missing tag

File: rover/source/docs/drive-preparation-and-operation.rst [hunk 8]
```diff
@@ -0,0 +1,11 @@
+==========
```
Comment (Hwurzburg) [inline rover/source/docs/drive-preparation-and-operation.rst:1]: missing tag

File: rover/source/docs/further-information.rst [hunk 9]
```diff
@@ -0,0 +1,10 @@
+==========
```
Comment (Hwurzburg) [inline rover/source/docs/further-information.rst:1]: missing tag

File: rover/source/docs/optional-hardware.rst [hunk 10]
```diff
@@ -0,0 +1,20 @@
+Optional hardware
```
Comment (Hwurzburg) [inline rover/source/docs/optional-hardware.rst:1]: missing tag and overline

File: rover/source/docs/optional-hardware.rst [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+=================
+
+Telemetry Radio
+---------------
```
Comment (Hwurzburg) [inline rover/source/docs/optional-hardware.rst:5]: header two is just an equals undelining   dashed is level three

File: rover/source/docs/ready-to-use-rovers.rst [hunk 12]
```diff
@@ -0,0 +1,4 @@
+Ready to Use Rovers
```
Comment (Hwurzburg) [inline rover/source/docs/ready-to-use-rovers.rst:1]: missing tag and overline

File: rover/source/docs/rover-components.rst [hunk 13]
```diff
@@ -1,14 +1,3 @@
-.. _gettit:
```
Comment (Hwurzburg) [inline rover/source/docs/rover-components.rst:1]: this removes tag and new title in missing overline

File: rover/source/docs/rover-variants.rst [hunk 14]
```diff
@@ -0,0 +1,13 @@
+==========
```
Comment (Hwurzburg) [inline rover/source/docs/rover-variants.rst:1]: missing tag

File: rover/source/docs/setup.rst [hunk 15]
```diff
@@ -0,0 +1,10 @@
+==========
```
Comment (Hwurzburg) [inline rover/source/docs/setup.rst:1]: missing tag

File: copter/source/docs/introduction.rst [hunk 16]
```diff
@@ -1,72 +1,67 @@
...
+Copter is a part of the broader ArduPilot software platform. 
+It works with a wide array of :ref:`Ground Control Station programs <common-choosing-a-ground-station>`, enabling vehicle setup, real-time flight monitoring, and robust mission planning. 
+As part of the larger ArduPilot ecosystem, Copter can be used with shared simulators, log analysis tools, and advanced APIs for vehicle control.
 
-ArduPilot is already a preferred platform for numerous commercially available autopilot systems but you can also use it to enhance the abilities of your own DIY multirotor.
+ArduPilot is the preferred platform for many commercially available autopilot systems and can significantly enhance DIY multirotor's capabilities.
 
-Key features
-============
+Key Features
```
Comment (Hwurzburg) [inline copter/source/docs/introduction.rst:19]: missing header underline using =====

File: copter/source/docs/introduction.rst [hunk 17]
```diff
@@ -1,72 +1,67 @@
...
+-  :ref:`Ad-hoc commands in Flight <ac2_guidedmode>`: If equipped with a two-way telemetry radio, simply click on the map, and the vehicle will fly to the selected location.
+-  :ref:`Autonomous missions <auto-mode>`: Design complex missions with hundreds of GPS waypoints using a ground station. Switch the vehicle to "AUTO", and it autonomously takes off, completes the mission, returns home, lands, and disarms without any human intervention.
+-  :ref:`Failsafes <failsafe-landing-page>`: The software continuously monitors the system's state and autonomously triggers a return-to-home if it loses contact with the pilot, encounters low battery, or strays outside a defined geofence.
+-  **Flexibility and Customizability**: Copter can fly :ref:`all shapes and sizes of vehicles <common-all-vehicle-types>` according to your preferences as it allows access to hundreds of parameters controlling its behavior.
+-  **No Vendor Lock-in**: ArduPilot is an entirely open-source platform backed by a diverse community of developers. It leaves you in total control of the software on your vehicle and its performance.
 
-Getting started
-===============
+Getting Started
```
Comment (Hwurzburg) [inline copter/source/docs/introduction.rst:33]: same here

File: copter/source/docs/introduction.rst [hunk 18]
```diff
@@ -1,72 +1,67 @@
...
+If you plan to build your own multicopter, the following pages will guide you. 
+Begin by understanding what a multicopter can do, and how to select a frame, autopilot board, and other essential components. 
+Then proceed to :ref:`First Time Setup <initial-setup>` to assemble your copter and :ref:`First Flight <flying-arducopter>` to configure and tune it.
 
-Learn more about Copter
-=======================
-
-To find out more about Copter and your main configuration decisions,
-please see the topics below:
+Discover More About Copter
```
Comment (Hwurzburg) [inline copter/source/docs/introduction.rst:52]: same here

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
a few more format things...

I also suggest you break this into two PR while you are setting up a working branch to facilitate review...1) the reorgs....and 2) the copter rewrites of content

**Comment by brianmusslewhite** [comment]
@Hwurzburg Sounds good, and thank you for all the feedback. I'll make sure to split it into a different branch next time and separate the PRs. I'll have the update PR soon.

How did you catch the errors that the Cl was missing?

**Comment by Hwurzburg** [comment]
by building the wiki locally...
I brought the PR down to my local machine repo with a fetch
then used update.py to build it (the FETCH_HEAD) locally and it outputs the errors and warnings as it builds...also allows to view the finished product

**Comment by brianmusslewhite** [comment]
Got it. I am building it locally but I must have missed the errors. I'll watch out for them.

**Comment by Hwurzburg** [comment]
hard to miss in my setup
Screenshot from 2023-07-09 12-50-17

**Comment by brianmusslewhite** [comment]
Sounds good. These changes should be pretty straightforward. 

What are your thoughts on the content, specifically the revised top-level menu titles? Do they make sense to you? Also, are there any additional category suggestions that come to mind?

Just to provide some context for these changes: my first encounter with the wiki felt a bit daunting and I was unsure where to start. My objective here is to reduce this intimidation factor for newcomers, while also improving the overall professional look-and-feel of the site.

**Comment by Hwurzburg** [comment]
I have not spent enough time reviewing in detail...but first impressions are that it goes a bit too far....some topics that are referred to often by users get too many clicks away....here is how I would compress Plane:

Introduction (use cases and flight features under here)
AutoPilot Hardware and Selection
First Time Setup (QuadPlane setup could go here...Quadplane needs all the fixed wing stuff setup first)
First Flight (Quadplane Tuning could go here)
Advanced Configuration  (maybe reorg that page with Mission Planning at the top, then everything else in a Advanced Param/Feature Section including Antenna Tracking)
Peripheral Hardware
If A Problem Arises (logs might go under here...people need to know this section is in the wiki...it needs to be expanded but...)
User Alerts (must be in left hand menu or even on the ArduPilot page...which can be condensed also)
Advanced Information (all other left hand topics)

I feel strongly that Autopilots, Adv Config and Periph Hdwr be in the left menu

**Comment by brianmusslewhite** [comment]
Closing pull request to split into two

**Pull request closed without merging**